flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStMapState.java

```diff
@@ -139,7 +140,8 @@ public byte[] serializeValue(UV value) throws IOException {
     public UV deserializeValue(byte[] valueBytes) throws IOException {
         DataInputDeserializer inputView = valueDeserializerView.get();
         inputView.setBuffer(valueBytes);
-        return userValueSerializer.deserialize(inputView);
+        boolean isNull = inputView.readBoolean();
```

Comment:
nit: could be one line and not introduce a variable

Reply:
That's not true.

when output, writing a single 'false' mean this is not null. 'true' means it is null.

This is a pair of `readBoolean` / `writeBoolean` for `inputView` / `outputView`, renaming the method is another topic.